rpcs3/rpcs3qt/save_data_list_dialog.cpp

```diff
@@ -158,10 +87,19 @@ save_data_list_dialog::save_data_list_dialog(const std::vector<SaveDataEntry>& e
 	LoadEntries();
 	UpdateList();
 
+	connect(m_list, &QTableWidget::cellChanged, [&](int row, int col) {
```

Comment:
I know it's bloated, but it's a bit complicated to transform from row, column to the entry name.  There's a lot of indirection.  So, I'm fine with having the extra variables.  If you want, I can optimize it to have less, but it reads better IMO as it is